Collaboration2/Collaboration2/Animations/Animation Views/CloudView.swift

```diff
@@ -26,6 +26,7 @@ struct CloudView: View {
                     CloudCell(startingAt: cloud.position, duration: cloud.duration, screenSize: geometry.size, size: size, imageName: imageName, direction: cloud.direction)
                 }
             }
+            .frame(maxWidth: .infinity, maxHeight: .infinity)
```

Comment:
remove this, the components have maxWidth and maxHeight always set to infinity by default, you should know better!!